packages/ontario-frontend-cli/bin/ontario-remove-package.js

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env node
+const Command = require('commander').Command;
+const Argument = require('commander').Argument;
+const { handleRemovePackageCommand } = require('../commands/ontario-remove-package/handleRemovePackage');
+const { ROOT_DIR } = require('../core/config');
+const { readPackageJson } = require('../core/utils');
+const logger = require('../core/utils/logger');
+const { doesPackageJsonExist, isOntarioFrontendProject } = require('../core/validation');
+
+const program = new Command();
+
+/**
+ * readPackageJson() is an async function that is used to read the contents of the package.json file.
+ * 
+ * These contents are checked against in several places in the below code, therefore the all of the 
+ * program commands should be wrapped in a self-executing async function.
+ * 
+ * Without this async wrapper function, the program will exit unexpectedly after user input.
+ */ 
+(
+  async () => {
+    const packageJson = async() => await readPackageJson(ROOT_DIR);
+
+    program
+      .name('ontario-remove-package')
+      .version(packageJson.version, '-v, --version', 'Output the current version');
+      // TODO: Add a description to package.json
+      // .description(packageJson.description);
+
+    program.addHelpText(
+      'after',
+      `\n"eslint" and "prettier" are the available packages to remove from your project. For more information, visit our developer site at https://developer.ontario.ca`,
+    );
+
+    program
+      .addArgument(new Argument('<package>', 'An Ontario package').choices(['eslint', 'prettier']))
```

Comment:
I know we have this as an argument in add-package as well, but I am trying to think about CLI UX for both at the same time. 

Currently this will require an argument to be passed:

```sh
mattstjean@macadmins-MacBook-Pro my-frontend-project % ontario-remove-package  
error: missing required argument 'package'
```

It would be beneficial for the argument to be optional in case the user just forgets what they can add or remove.  This would require, though, that we then prompt them for the package name(s). 

It would be beneficial for the argument to be variadic too so that the user can specify any number of packages to be removed. I can't think of a reason we need to limit this process to one at a time. 

https://www.npmjs.com/package/commander#commands

We are also tied tightly to spelling, learning the correct spelling after getting it wrong once, could we parse the argument and normalize it to improve the experience? There's some examples of this:

```js
  .addOption(new Option('--donate [amount]', 'optional donation in dollars').preset('20').argParser(parseFloat))
```

The documentation doesn't talk too much about arg parsing and whatnot but you can see how to set it up from the comment and their Argument code

https://www.npmjs.com/package/commander?activeTab=code

Reply:
@matt-stjean I think this would be a valuable improvement. Without changing scope to much, could we capture these improvements within another ticket?

Reply:
I think that is a good idea since this PR is following precedent set by add-package, and I agree that it would be expanding scope.

I will put this in a ticket.